applications/templates/observe/kube-ops-view.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kube-ops-view
+spec:
+  project: observe-project
```

Comment:
it is implicitly a project, why sub call it project

I missed this when you were doing it. 

you would not call the application name `kube-ops-view-application`

Reply:
At the beginning I added that prefix for clarity, but I do agree that it doesn't make sense anymore. In this PR I've added it just for consistency and I was planning to make a separate PR to remove them all at once